packages/webapp/src/pages/delegates/ui/index.tsx

```diff
@@ -0,0 +1,77 @@
+import { GetServerSideProps } from "next";
+import { QueryClient, useQuery } from "react-query";
+import { dehydrate } from "react-query/hydration";
+import { BsArrowDown } from "react-icons/bs";
+
+import { Container, FluidLayout, Heading, queryMembers, Text } from "_app";
+import { DelegateChip } from "elections";
+import { MemberData } from "members/interfaces";
+
+export const getServerSideProps: GetServerSideProps = async ({ query }) => {
+    const queryClient = new QueryClient();
+    return {
+        props: {
+            dehydratedState: dehydrate(queryClient),
+        },
+    };
+};
+
+interface Props {
+    delegatesPage: number;
+}
+
+const MEMBERS_PAGE_SIZE = 4;
```

Comment:
I don't think this is relevant for delegate. If it's related to the logged user we will not have the logged user session in our server, so hydrating it here does not make sense. (Unless we plan to implement user sessions handling in the server)

Reply:
tldr; this block can be removed and we can let the client execute the query as it's doing below

Reply:
I'll remove the boilerplate for now. I erred on the side of leaving stuff in for us to remove when we hook this UI up vs. making everything perfectly clean. We are going to need to decide what we want on this page when a member is not logged in. If, for example, we want to show the Head Chief / Satoshi on this page, that could potentially be SSR'd.